### DISCUSSION TITLE
Dịch "Pseudoinverse"

### DISCUSSION THREAD

**Discussion by quangnhat185** [mlbvn/handson-ml2-vn#22] [Terminology] [answered]
Trong phần mình dịch có thuật ngữ `Pseudoinverse` ý chỉ dạng tổng quát hóa của một ma trận nghịch đảo. Hiện tại mình đang dịch từ này là `ma trận nghịch đảo tổng quát` nhưng thấy hơi dài. Mọi người cho ý kiến nhé?

cc: @mlbvn/handson-ml2 

__Context__:
![image](https://user-images.githubusercontent.com/41325216/107874208-3492b600-6eb8-11eb-9ca6-48fb70a14664.png)

__Reference__: [Generalized inverse Wikipedia](https://en.wikipedia.org/wiki/Generalized_inverse)

**Comment by rootonchair** [comment, marked as answer]
FYI, em thấy nó những tên gọi khác như:
* Ma trận giả đảo
* Ma trận giả nghịch đảo

**Reply by tiepvupsu** [reply]
Từ  này đứng riêng thì gọi là `giả nghịch đảo` nhé. `Giả nghịch đảo` đã được dùng phổ biến rồi.
